From b61067859112648bc858aa2a1fc72bc90374433e Mon Sep 17 00:00:00 2001
From: Dmitry Klochkov <dmitry.klochkov@bell-sw.com>
Date: Fri, 20 Oct 2023 13:14:51 +0300
Subject: [PATCH] return back support for some architectures

This is needed to make tests/abuild_test:abuild_deps pass.
---
 functions.sh.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/functions.sh.in b/functions.sh.in
index 82cacfc..602de56 100644
--- a/functions.sh.in
+++ b/functions.sh.in
@@ -7,6 +7,9 @@ program=${0##*/}
 arch_to_hostspec() {
 	case "$1" in
 	aarch64)	echo "aarch64-$VENDOR_DISTRO-linux-@libc@" ;;
+	armhf)		echo "armv6-$VENDOR_DISTRO-linux-@libc@eabihf" ;;
+	ppc64le)	echo "powerpc64le-$VENDOR_DISTRO-linux-@libc@" ;;
+	s390x)		echo "s390x-$VENDOR_DISTRO-linux-@libc@" ;;
 	x86_64)		echo "x86_64-$VENDOR_DISTRO-linux-@libc@" ;;
 	*)		echo "unknown" ;;
 	esac
@@ -15,6 +18,9 @@ arch_to_hostspec() {
 hostspec_to_arch() {
 	case "$1" in
 	aarch64*-*-*-*)		echo "aarch64" ;;
+	armv6*-*-*-*eabihf)	echo "armhf" ;;
+	powerpc64le-*-*-*)	echo "ppc64le" ;;
+	s390x-*-*-*)		echo "s390x" ;;
 	x86_64-*-*-*)		echo "x86_64" ;;
 	*)			echo "unknown" ;;
 	esac
-- 
2.42.0

